### ISSUE TITLE
[feature request] profiles

### ISSUE THREAD

**Issue by cocoonkid** [Zuntek/WeekToDoWeb#142]
As a user I would like to be able to switch between profiles.

For example one for work and one for family.

This would allow me to have everything kept in sync (syncthing). Currently I am using family instances on laptops and work instances on desktops.

Alternatively as a workaround I sill try to run multiple instances but I can already smell these will be pretty whacky hacks :-)

**Comment by abdallahkorraim** [comment]
@manuelernestog, yes that would be nice, workspaces or profiles may also act as different projects
